local/localSystem.ini

```diff
@@ -0,0 +1,6 @@
+[Parent_Dir]
```

Comment:
It would probably be a good idea to add comments to all of these settings.

I also wonder if we should name the file differently if we don't anticipate using it for purposes other than configuration management -- e.g. something like ConfigLocations.ini.

Reply:
Renamed into DirLocations.ini because it configures a parent local directory and could be used for more than just configs. 

Also added comments.